crates/collab/src/api/events.rs

```diff
@@ -1442,65 +1442,64 @@ fn for_snowflake(
             Event::App(e) => {
                 let mut properties = json!({});
                 let event_type = match e.operation.trim() {
-                    "extensions: install extension" => "Extension Installed".to_string(),
+                    // App
                     "open" => "App Opened".to_string(),
-                    "project search: open" => "Project Search Opened".to_string(),
-                    "first open" => {
-                        properties["is_first_open"] = json!(true);
```

Comment:
This is an example of adding a value to the properties that seems meaningless, the app name itself is what implies it is the first app open. I did similar removals for things being stuffed into properties below.